recipes/dash-labs/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "dash-labs" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # v0.2.0 sdist on pypi is invalid and doesn't include requirements.txt
+  # Install from github archive
+  url: https://github.com/plotly/dash-labs/archive/v{{ version }}.tar.gz
+  sha256: 7f7b77881476059206e45e38360afe34ab3b5ff2e2ca94a90ffd6ae239b1970c
+build:
+  number: 0
+  skip: true   # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dash >=1.18.1
+    - dash-bootstrap-components >=0.11.1
+    - dataclasses ==0.8  # [py<37]
```

Conda forge has a noop version of dataclasses for python37+
```suggestion
    - dataclasses ==0.8 
```